web-server/src/utils/dora.ts

```diff
@@ -6,6 +6,23 @@ import {
 } from 'date-fns/constants';
 import { isNil, mean, reject } from 'ramda';
 
+export enum IndustryStandardsDoraScores {
```

Comment:
there should be an enum of all values like:
```
export enum Industries = {
ALL_INDUSTRIES = "ALL_INDUSTRIES"
...
}
```

and then a map like 
```
IndustryStandardsDoraScores = {
[Industries.ALL_INDUSTRIES] : 6.3,
...
}
```

that way we can easily iterate over this entire set

Reply:
Great idea.
Addressed this in https://github.com/middlewarehq/middleware/pull/499